mappings/net/minecraft/server/rcon/BufferHelper.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS none/nc net/minecraft/server/rcon/BufferHelper
+	FIELD a HEX_CHARS [C
```

Comment:
`HEX_LOOKUP`? Because name things after what they're used for, not what they are?
(I don't mean to talk down or anything. Sorry if I do make that impression.)